subprojects/core/src/main/java/org/gradle/internal/model/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@NonNullApi
```

Comment:
💅 Don't use imports here per: https://bt-internal-docs.grdev.net/gbt/how-to/develop/nullability-guide/?h=%40nonn#annotate-packages

Reply:
Fixed. Though it would be good to have a checkstyle check for this and fix the many other occurrences. Has this recommendation been communicated at some point, since I didn't know about it?

Reply:
I don't know.  I remember commenting on the PR to ask about this, so I always remember this recommendation.

Reply:
Checkstyle and standardizing this is a good idea.